assets/css/forms.css

```diff
@@ -129,12 +129,33 @@ button:hover {
 /****************/
 
 .contact__main {
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-  justify-items: center;
-  margin: 8rem 6rem 4rem;
-  padding: 1rem;
-  min-height: 600px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 800px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
```

Comment:
- we replaced 'grid' with 'flex',

- we added a min-height so that our footer doesn't come up too far on the page, 

- we added max-width so that even on ultra-wide screens we keep the shape we wanted 
(this makes sure that our content width never goes above 1200px, this is useful for ultra-wide screens and makes sure our elements don't look displaced or a bit silly being too far apart)

- we added a margin: 0 auto; so that our content is always centered 
(this is the part that gives us the whitespace I mentioned above if the screen is ultra-wide)

Reply:
oh okay. the content in the page is less. so that why I thought no need to scrolling and did that. but, forgot about ultra-wide monitors 😅😬